src/main/sensors/voltage.c

```diff
@@ -226,8 +226,8 @@ void voltageMeterESCInit(void)
 {
 #ifdef USE_ESC_SENSOR
     memset(&voltageMeterESCState, 0, sizeof(voltageMeterESCState_t));
-    pt1FilterInit(&voltageMeterESCState.filter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatLpfPeriod), HZ_TO_INTERVAL(200)));
-    pt1FilterInit(&voltageMeterESCState.sagFilter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatSagPidCompPeriod), HZ_TO_INTERVAL(200)));
+    pt1FilterInit(&voltageMeterESCState.filter, pt1FilterGain(GET_BATTERY_LPF_FREQUENCY(batteryConfig()->vbatDisplayLpfPeriod), HZ_TO_INTERVAL(200)));
```

Comment:
should be changed back to 50hz here

Reply:
I think that has to match the battery sampling task rate.

Reply:
Thanks, missed that!

Reply:
ah, but it's the esc filter. My mistake.

Reply:
No, actually the battery sample task is at 200hz, so they should have been 200, reverting back to 200.

Reply:
yes, my mistake, sorry